Package.swift

```diff
@@ -1,8 +1,9 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.3
 import PackageDescription
 
 let package = Package(
     name: "CareKit",
+    defaultLocalization: "en",
```

Comment:
May not actually need this. I added while debugging

Reply:
I believe that is indeed required!